src/sentry/utils/committers.py

```diff
@@ -255,7 +255,7 @@ def get_event_file_committers(
     munged = munged_filename_and_frames(event_platform, frames, "munged_filename", sdk_name)
     if munged:
         frames = munged[1]
-    app_frames = [frame for frame in frames if frame.get("in_app")][-frame_limit:]
+    app_frames = [frame for frame in frames if frame and frame.get("in_app")][-frame_limit:]
```

Comment:
Another option would be to remove None from the result of https://github.com/getsentry/sentry/blob/master/src/sentry/utils/event_frames.py#L209 but I was having trouble setting up a test for this, and ultimately they do the same thing.